Makefile

```diff
@@ -27,16 +26,15 @@ test: $(shunit2)
 
 .PHONY: install
 install: $(include_directory)
-	$(INSTALL) $(source_path) $(target_path)
 	$(INSTALL) shell-includes/error.sh shell-includes/usage.sh shell-includes/variables.sh shell-includes/warning.sh $(include_directory)
-	$(SED) --in-place --expression='s/\(\.\/\)\?$(source_file)/$(target_file)/g' $(target_path)
-	$(SED) --in-place --expression='s#^\(includes=\).*#\1"$(include_directory)"#' $(target_path)
+	$(INSTALL) $(source_path) $(target_path)
+	$(SED) 's#^\(includes=\).*#\1"$(include_directory)"#' $(source_path) > $(target_path)
 
 $(shunit2):
 	$(CURL) --silent --location "https://github.com/kward/shunit2/archive/source.tar.gz" | tar --extract --gzip
 
 $(include_directory):
-	$(MKDIR) $(include_directory)
+	mkdir $(include_directory)
```

Comment:
If `mkdir` is somewhere else you can achieve the same effect by defining `MKDIR=/path/to/mkdir` when running `make`. Having a variable for every command except `rm` is considered best practice to allow running `make` no matter what `PATH` is set to.